transport/x/grpc/util_test.go

```diff
@@ -66,3 +67,21 @@ func TestProcedureNameFunctions(t *testing.T) {
 		})
 	}
 }
+
+func TestProcedureNameEmpty(t *testing.T) {
+	_, _, err := procedureNameToServiceNameMethodName("")
+	require.Error(t, err)
+	_, err = procedureNameToFullMethod("")
+	require.Error(t, err)
+}
+
+func TestToFullMethodEmptyServiceName(t *testing.T) {
+	require.Equal(t, "/__default__/foo", toFullMethod("", "foo"))
```

Comment:
A reminder that I remain not a fan of this `__default__/name` scheme, but not a blocker. I would like to hear about alternatives you’re discarding and why. Do we have an opportunity to make a better decision here if we pursue @willhug’s work on bypassing gRPC’s encoding and route table equivalents?